aodncore/pipeline/watch.py

```diff
@@ -211,8 +221,15 @@ def _register_tasks(self):
                     "{name} not in {handlers}".format(name=items['handler'], handlers=available_handler_names))
 
             params = items.get('params', {})
-            task_object = self._build_task(pipeline_name, handler_class, params)
-            self._application.register_task(task_object)
+
+            try:
+                _ = handler_class('', config=self._config, **params)
+            except TypeError as e:
+                warnings.warn("invalid parameters for handler '{name}': {e}".format(name=items['handler'],
+                                                                                    e=format_exception(e)))
```

Comment:
This warning appears in the watchservice log, and in every task's stderr.log, and there's currently no indication which one has the bad config. Could we add the task name in the message?

Better still, is it possible to have this appear only in the badly configured task's log?

Reply:
> This warning appears in the watchservice log, and in every task's stderr.log, and there's currently no indication which one has the bad config. Could we add the task name in the message?

Yes, this is an oversight, fixing.

> Better still, is it possible to have this appear only in the badly configured task's log?

Not without a deeper refactor of the way the workers work.

The way it is, each worker is *aware* of the other tasks, but only picks up tasks that it is supposed to. So each task *sees* the errors but they aren't actually a problem for that pipeline. Given that it will be picked up in RC, I think we can leave this enhancement until later!

Reply:
:+1: